src/ui/functions/buildLegend.ts

```diff
@@ -13,8 +13,9 @@ export const buildLegend = (store: NamespacedStore) => {
   const colors = store.select("cat_viz").mat_colors;
   const mat = store.select("network").mat;
   const flatMat = flatten<number>(mat);
```

Comment:
Do we know how big this flatMat is? Just curious what size this got to trigger a stack size error on Math.min

Reply:
I think it was triggered in 500x500 matrix